Documents/BallotDrafts/sarif-v2.1.0-final-e-ballot-1/sarif-v2.1.0-final.htm

```diff
@@ -8720,7 +8720,7 @@ <h3><a name="_Toc33187404"></a><a name="_Ref525810085">3.16.4 guid property</a><
 of <span class=CODEtemp>theSarifLog.inlineExternalProperties</span> (�3.13.5) and <span class=CODEtemp>location</span> (�3.16.3) is absent, then <span
 class=CODEtemp>guid</span> <b>SHALL</b> be present.</p>
 
-<p class=MsoNormal>Otherwise (that is, if the extermalized properties are
+<p class=MsoNormal>Otherwise (that is, if the externalized properties are
```

Comment:
Again, this is hopefully editorial, but if it isn't, if there's an errata process or at least a way to have future versions not have content like this...